src/Models/restaurants.json

```diff
@@ -0,0 +1,141 @@
+[  
+    {
+        "restaurantName": "MBM Trinbago Meals",
+        "address": "Pool Road, Birmingham B66 3DB",
+        "phoneNumber": {
+            "landline": null, 
+            "mobile": "07792557204"
+        },
+        "cusineCategory": "Trinidadian, dine-in and takeaway", 
+        "tripAdvisorRating": 5, 
+        "website": null, 
+        "instgram": null,
+        "facebook": "@mbmmeals", 
+        "email":"melinda.williams@live.co.uk", 
+        "imageLinks": {
+            "facebookImages": "https://www.facebook.com/mbmmeals/photos/a.111542720538174/141008090924970/?type=3&theater",
+            "instagramImages": null
+        } 
+    },
+
+    {
+        "restaurantName":"Earth's Kitchen",
+        "address": "180 High Street Deritend Digbeth, Birmingham, B12 0LD",
+        "phoneNumber": {
+            "landline": "0121 572 6163", 
+            "mobile": null
+        },
+        "cusineCategory": "Plant based Caribbean, dine-in and takeaway", 
+        "tripAdvisorRating": 5, 
```

Comment:
not really a comment but was wondering if anyone knows if trip advisor has an API where we could create a get request where it searches for a name and location match then automatically returns the rating. Users leaving our app to go look at trip advisor maybe an issue.

Reply:
I had a look and TripAdvisor have an API for developers called the Content API. 
http://developer-tripadvisor.com/content-api/ 

They also have widgets:  https://www.tripadvisor.com/Widgets

Reply:
ooh that is good to know! thanks for looking.